src/control/layer/LayerController.cpp

```diff
@@ -279,9 +273,7 @@ void LayerController::switchToLay(int layer, bool hideShow) {
     p->setSelectedLayerId(layer);
 
     if (hideShow) {
-        for (size_t i = 1; i <= p->getLayerCount(); i++) {
-            p->setLayerVisible(i, static_cast<int>(i) <= layer);
-        }
+        for (size_t i = 1; i <= p->getLayerCount(); i++) { p->setLayerVisible(i, static_cast<int>(i) <= layer); }
```

Comment:
Four similar unnecessary style changes in this file.

Reply:
Please revert this change.